inc/Ui/Diff.php

```diff
@@ -95,27 +94,33 @@ protected function handle()
     {
         global $INPUT;
 
-        // difflink icon click, eg. ?rev=123456789&do=diff
+        // diff link icon click, eg. &do=diff&rev=#
         if ($INPUT->has('rev')) {
```

Comment:
```suggestion
        // $REV retrieved from $INPUT->has('rev') and filtered in @see pageinfo()
        if ($REV) {
```

Previous version of DokuWiki uses global $REV as rev1. Which is set in doku.php, cleanup in pageinfo() to `''`. Therefore, using `?do=diff&rev=<timestamp of the current wikipage>`, results in comparing the last revision with current page.

Reply:
it needs to support `do=diff&rev=<current revision number>` for both page and media. I added fallback to compare previous with current in Ui\Diff.php 4cf9cfa